node/cmd/guardiand/node.go

```diff
@@ -458,6 +464,12 @@ func init() {
 	xrplEvmRPC = node.RegisterFlagWithValidationOrFail(NodeCmd, "xrplEvmRPC", "XRPLEVM RPC_URL", "ws://eth-devnet:8545", []string{"ws", "wss"})
 	xrplEvmContract = NodeCmd.Flags().String("xrplEvmContract", "", "XRPLEVM contract address")
 
+	plasmaRPC = node.RegisterFlagWithValidationOrFail(NodeCmd, "PlasmaRPC", "PLASMA RPC_URL", "ws://eth-devnet:8545", []string{"ws", "wss"})
+	plasmaContract = NodeCmd.Flags().String("plasmaContract", "", "Plasma contract address")
+
+	creditCoinRPC = node.RegisterFlagWithValidationOrFail(NodeCmd, "CreditCoinRPC", "CREDITCOIN RPC_URL", "ws://eth-devnet:8545", []string{"ws", "wss"})
```

Comment:
for consistency, should be `creditCoinRPC` for the argument

```suggestion
	creditCoinRPC = node.RegisterFlagWithValidationOrFail(NodeCmd, "creditCoinRPC", "CREDITCOIN RPC_URL", "ws://eth-devnet:8545", []string{"ws", "wss"})
```